src/zdns/lookup.go

```diff
@@ -995,7 +996,12 @@ func (r *Resolver) extractAuthority(ctx context.Context, authority interface{},
 			q.Q.Type = dns.TypeA
 		}
 		q.RetriesRemaining = &r.retriesRemaining
+
+		// A/AAAA records for NSes are not on the chain of trust, so we don't need to validate DNSSEC
+		// Doing this to save us some time (this can propogate A LOT of queries in certain cases)
+		r.shouldValidateDNSSEC = false
 		res, trace, status, _ = r.iterativeLookup(ctx, &q, r.rootNameServers, depth+1, ".", trace)
+		r.shouldValidateDNSSEC = true
```

Comment:
Do we want to re-enable this in all cases? What if we're not validating DNSSEC?

Reply:
Good catch! 989055717374f7f39a108b4cbad318596ed4acb9